service/src/main/java/bio/terra/workspace/app/configuration/external/SamConfiguration.java

```diff
@@ -11,11 +12,21 @@ public class SamConfiguration {
   /** URL of the SAM instance */
   private String basePath;
 
+  private Duration permissionsCacheTtl;
+
   public String getBasePath() {
     return basePath;
   }
 
   public void setBasePath(String basePath) {
     this.basePath = basePath;
   }
+
+  public Duration getPermissionsCacheTtl() {
```

Comment:
What does Ttl stand for?

Reply:
Having read the rest of the PR, it's the length of the length of time the cache is good for. So… "time to ??"?

Can we change this to something more self-documenting, like "duration"?

Reply:
Time to live. Maybe `permissionsCacheLifetime` would be clearer?